nservicebus/hosting/override-hostid.md

```diff
@@ -1,27 +1,29 @@
 summary: How to override the endpoint host identifier
 component: Core
 versions: '[4.0,)'
 redirects:
  - nservicebus/override-hostid
+reviewed: 2016-10-19
 ---
 
 
 ## Host identifier
 
-In NServiceBus Versions 4.4 and above, all messages sent to the audit queue include two extra headers, these are `$.diagnostics.hostid` and `$.diagnostics.hostdisplayname`. These extra headers uniquely identify the running host (not to be confused with `NServiceBus.Host`, in this scenario host refers to the operating system host) for the endpoint. The defaults are the machine name for `$.diagnostics.hostdisplayname` and for `$.diagnostics.hostid` is a hash of the running executable installed path concatenated with the machine name. These defaults mostly work, except in environments where endpoint upgrades are done to a new path or in Azure deployments.
+In NServiceBus, all messages sent to the audit queue include two extra headers, these are `$.diagnostics.hostid` and `$.diagnostics.hostdisplayname`. These extra headers uniquely identify the running host for the endpoint, i.e. the operating system host (not to be confused with `NServiceBus.Host`). 
 
+The host ID is used by ServiceControl to map a running endpoint to the host where they are deployed. This information is then displayed in ServicePulse and ServiceInsight so it's possible to identify on which host the endpoint is running.
 
-## Host identifier usage
+The default values for `$.diagnostics.hostdisplayname` is the machine name and for `$.diagnostics.hostid` is a hash of the running executable's  path concatenated with the machine name. These defaults work in most scenarios, except in environments where endpoint upgrades are done to a new path or in Azure deployments.
 
-The host ID is used by ServiceControl to map a running endpoint to the host where they are deployed. This information is then displayed in ServicePulse so that OPS knows the "host" that an endpoint is deployed to, so they can quick diagnose issues with the host environment.
 
+## Overriding the host identifier
 
-## Override an endpoint host identifier
+There are scenarios where the default rules used to generate a `hostid` and `hostdisplayname` are not adequate and the user needs to take control, i.e. in environments where endpoint upgrades are done to a new path or in Azure deployments.
 
-As described above, there are scenarios where the default rules used to generate a `hostid` and `hostdisplayname` are not adequate and the user needs to take control. In an Azure deployment the NServiceBus framework takes care of updating these defaults for the user automatically, so in Azure the default are role name for `$.diagnostics.hostdisplayname` and for `$.diagnostics.hostid` is the role InstanceId.
+In an Azure deployment, the NServiceBus framework takes care of updating these defaults for the user automatically, i.e. the `$.diagnostics.hostdisplayname` defaults to the role name and the `$.diagnostics.hostid` contains the instance ID.
```

Comment:
@timbussmann I'm a bit confused, since Azure does that automatically why don't we say "this is default for Azure" instead of talking about the manual control? In fact from the perspective of the user this is also happening automatically, correct?

/cc: @SeanFeldman 

Please, clarify, then I'll merge :)